.github/workflows/latest_snapshot_push.yml

```diff
@@ -1,12 +1,14 @@
 name: Build latest OS and EE image
 
 on:
-  push:
-    branches:
-      - master
+  workflow_dispatch:
```

Comment:
Who will trigger the snapshot build?

Reply:
After this PR passes, @hasancelik and I are planning to do it together. It will be triggered by Jenkins release pipeline.

Reply:
Afaik, it should be triggered by `Hazelcast-4.master-deploy` jenkins job, not release pipeline.

Reply:
We changed the triggerin Jenkins job to  `Hazelcast-docker-snapshot-release`.